invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookTab/BookTab.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import {
+  Icon,
+  Tab,
+  Accordion,
+  Segment,
+  Menu,
+  Grid,
+  Responsive,
+} from 'semantic-ui-react';
+
+class BookAccordion extends Component {
+  state = { activeIndex: 0 };
+
+  handleClick = (e, titleProps) => {
+    const { index } = titleProps;
+    const { activeIndex } = this.state;
+    const newIndex = activeIndex === index ? -1 : index;
+
+    this.setState({ activeIndex: newIndex });
+  };
+
+  render() {
+    const { activeIndex } = this.state;
+
+    return (
+      <Accordion fluid styled>
+        <Accordion.Title
+          active={activeIndex === 0}
+          index={0}
+          onClick={this.handleClick}
+        >
+          <Icon name="dropdown" />
+          Abstract
+        </Accordion.Title>
+        <Accordion.Content active={activeIndex === 0}>
+          <p>{this.props.data.abstracts[0]}</p>
+        </Accordion.Content>
+
+        <Accordion.Title
+          active={activeIndex === 1}
+          index={1}
+          onClick={this.handleClick}
+        >
+          <Icon name="dropdown" />
+          Information
+        </Accordion.Title>
+        <Accordion.Content active={activeIndex === 1}>
+          <p>{this.props.data.keywords}</p>
+        </Accordion.Content>
+
+        <Accordion.Title
+          active={activeIndex === 2}
+          index={2}
+          onClick={this.handleClick}
+        >
+          <Icon name="dropdown" />
+          Chapters
+        </Accordion.Title>
+        <Accordion.Content active={activeIndex === 2}>
+          <p>{this.props.data.chapters[0]}</p>
+        </Accordion.Content>
+      </Accordion>
+    );
+  }
+}
+
+class BookTabMenu extends Component {
```

Comment:
In its own file too, you can group them together in a module if you like since BookTabMenu and BookTabs are combined to create a complex component.